src/Home/_home.scss

```
@@ -1,4 +1,14 @@
 //index
+.h1{
```

```suggestion
h1 {
```

`.h1` implies that the class name is h1, which it is not and should **NEVER** be